test.js

```diff
@@ -30,8 +30,8 @@ util.inherits(MyPersistence, CachedPersistence)
   'outgoingUpdate', 'outgoingClearMessageId',
   'incomingStorePacket', 'incomingGetPacket',
   'incomingDelPacket', 'putWill', 'delWill',
-  'createRetainedStream', 'outgoingStream',
-  'subscriptionsByClient',
+  'createRetainedStream', 'createRetainedStreamCombi',
```

Comment:
[this](https://github.com/mcollina/aedes-cached-persistence/pull/4/files#diff-168726dbe96b3ce427e7fedce31bb0bcR143) should make it pass.